src/CADL.Extension/Emitter.Csharp/package.json

```diff
@@ -50,17 +50,17 @@
     },
     "dependencies": {
         "@autorest/csharp": "3.0.0-beta.20230309.1",
-        "@azure-tools/typespec-autorest": "0.29.0",
-        "@azure-tools/typespec-azure-core": "0.29.0",
-        "@azure-tools/typespec-client-generator-core": "0.30.0-dev.13",
-        "@typespec/compiler": "0.43.0",
-        "@typespec/eslint-config-typespec": "0.6.0",
-        "@typespec/eslint-plugin": "0.43.0",
-        "@typespec/http": "0.43.1",
-        "@typespec/library-linter": "0.43.0",
-        "@typespec/openapi": "0.43.0",
-        "@typespec/rest": "0.43.0",
-        "@typespec/versioning": "0.43.0",
+        "@azure-tools/typespec-autorest": "0.30.0",
```

Comment:
Can we move to 'use peerDependencies with a range'?  Is there any blocker in dotnet?

Reply:
You may refer https://github.com/Azure/autorest.java/blob/main/typespec-extension/package.json#L47-L53

Reply:
We have not make sure peerDependencies works for our project.  We will not implement `use peerDependencies with a range` in this PR.

Reply:
I see. I won't block the merge of this PR, but please create an issue to track the work of enabling PeerDependency.